### PR TITLE
Add missing section and adjust grammar

### PR DESCRIPTION
**Pull request by ludeeus**
## Proposed change

The "Tilgang til data" section was missing.
And a few grammatical issues were adjusted.

Adding a link to the PR that added the integration to make the bot happy <https://github.com/home-assistant/core/pull/107405>

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/elvia.markdown [hunk 1]
```diff
@@ -13,7 +13,7 @@ ha_integration_type: integration
 ---
 
 The Elvia integration will import your historical grid consumption.
-You can use the Elvia integration, regardless of which provider you are subscribed to as long as Elvia operates the grid, if you are unsure about if this can work for you. Login to [Min side](https://www.elvia.no/logg-inn/) if you see a meter there, you can use it.
+You can use the Elvia integration, regardless of which provider you are subscribed to, as long as Elvia operates the grid. If you are unsure, this can work for you. Login to [Min side](https://www.elvia.no/logg-inn/). If you see a meter there, you can use it.
```
Comment (ludeeus) [inline source/_integrations/elvia.markdown]:
```suggestion
You can use the Elvia integration, regardless of which provider you are subscribed to, as long as Elvia operates the grid. If you are unsure if this works for you, log in to [Min side](https://www.elvia.no/logg-inn/) and check. If you see a meter there, you can use it.
```

File: source/_integrations/elvia.markdown [hunk 2]
```diff
@@ -24,22 +24,22 @@ To use this integration, you need to create an API token.
...
 {% include integrations/config_flow.md %}
 
 ## Updates
 
 The integration will fetch new data every hour.
-The first time it runs, the data for the past 3 years will be imported.
+The first time it runs, the data for the past three years will be imported.
 
-The consumption data that is shown both on [Min side](https://www.elvia.no/logg-inn/) and in Elvia's app is the same data as this integration will use. And until the data is updated in those dashboards (which happens once every hour), the data will not be available in Home Assistant.
+The consumption data shown on [Min side](https://www.elvia.no/logg-inn/) and in Elvia's app is the same as this integration will use. And until the data is updated in those dashboards (which happens once every hour), the data will not be available in Home Assistant.
```
Comment (ludeeus) [inline source/_integrations/elvia.markdown]:
```suggestion
The integration is scheduled to be updated hourly, so there will be some delay when you see the updated data inside Home Assistant compared to [Min side](https://www.elvia.no/logg-inn/) and Elvia's mobile apps.
```

File: source/_integrations/elvia.markdown [hunk 3]
```diff
@@ -24,22 +24,22 @@ To use this integration, you need to create an API token.
...
 
 The integration will fetch new data every hour.
-The first time it runs, the data for the past 3 years will be imported.
+The first time it runs, the data for the past three years will be imported.
 
-The consumption data that is shown both on [Min side](https://www.elvia.no/logg-inn/) and in Elvia's app is the same data as this integration will use. And until the data is updated in those dashboards (which happens once every hour), the data will not be available in Home Assistant.
+The integration is scheduled to be updated hourly, so there will be some delay when you see the updated data inside Home Assistant compared to [Min side](https://www.elvia.no/logg-inn/) and Elvia's mobile apps.
 
-If you are after instant monitoring of your consumption you would have to use another integration, like [Tibber](/integrations/tibber/).
+If you are after instant monitoring of your consumption, you would have to use another integration, like [Tibber](/integrations/tibber/).
```
Comment (c0ffeeca7) [inline source/_integrations/elvia.markdown]:
```suggestion
If you are interested in real-time monitoring, this integration is not for you. In this case, you need to use a real-time metering integration like [Tibber](/integrations/tibber/).
```

**Review by c0ffeeca7 — COMMENTED** [review]
tweak

File: source/_integrations/elvia.markdown (hunk 3 above)
Comment (ludeeus) [reply source/_integrations/elvia.markdown]: Thanks 👍

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @ludeeus :+1:

**Pull request merged**